server/api/v2/openapi.json

```diff
@@ -237,6 +237,71 @@
         }
       }
     },
+    "/cluster/{clusterId}/upgrade-risks-prediction": {
```

Comment:
it's `clusters` with plural form in the rest of our api

Reply:
Not in v2 endpoints. You can check the "cluster/{cluster}/report" or "cluster/{cluster}/info" endpoints